frontend/app/src/hooks/use-organizations.ts

```diff
@@ -28,6 +30,36 @@ export function useOrganizations() {
     [organizationListQuery.data?.rows],
   );
 
+  // Fetch detailed organization data for each organization to get tenant status info
+  const detailedOrgQueries = useQuery({
+    queryKey: [
+      'organizations:detailed',
+      organizations.map((org) => org.metadata.id).sort(),
+    ],
+    queryFn: async () => {
+      const orgPromises = organizations.map(async (org) => {
+        try {
+          const result = await cloudApi.organizationGet(org.metadata.id);
+          return result.data;
+        } catch (error) {
+          console.warn(
+            `Failed to fetch detailed org data for ${org.metadata.id}:`,
+            error,
+          );
+          return null;
+        }
+      });
+      const results = await Promise.all(orgPromises);
+      return results.filter((org): org is Organization => org !== null);
```

Comment:
do you think we should have a bulk endpoint for this instead so we don't need to make N API calls here?